### ISSUE TITLE
Major List Update

### ISSUE THREAD

**Issue by oliviaklohr** [remoteintech/remote-jobs#772]
Hey all - 

I was working on compiling my own remote-first / remote-available tech company list when I stumbled across yours. After comparing the two, I have around ~160 companies on my list that aren't on yours (and you have way more that I don't, haha).

I was going to start work on compiling a PR in your format for the companies I have - which will take some time as your format is more in-depth - but I noticed in your PR Template that you were wanting the contributions to be opened by an employee of the company.

I wanted to verify that this type of contribution would be welcome before I put the work into doing so.

Thanks, and thanks for all your work on this project!

**Comment by nylen** [comment]
Hey @oliviaklohr, contributions of this type are very welcome. Thank you!

**Comment by oliviaklohr** [comment]
Absolutely! I'll hop on it and hopefully have it up in the near future. 😊

**Comment by brh** [comment]
@oliviaklohr could you possibly just dump your list somewhere like in a gist or pastebin for people to view before the merge, please???

Thank you

**Comment by oliviaklohr** [comment]
@brh yep! here's [a gist](https://gist.github.com/oliviaklohr/cb610c14ffc7f04336a622c7e7e16ec6) with just company names and a majority of the links to their sites. I've been working on compiling the list for a while, but I still need to go back through and double-check that these companies are still either remote-first or remote-allowing, not just doing it temporarily because of COVID, but I think I've done a fairly good job parsing those out.

**Comment by Hanee-Solanki** [comment]
Hi, 

I was hoping to contribute to this and this is my first time doing an open source project. Please advise if I could use the same gist or if there have been any changes to it.

Thank you very much!

**Comment by dougaitken** [comment]
Hi @Hanee-Solanki theres been no changes since the last comment so I believe you can use that list as a basis. Thanks.

**Comment by jainendra001** [comment]
Hi @dougaitken can I work on this issue, I am a full stack developer from India and currently an undergrad at IIT . Could you please assign it to me?

**Comment by dougaitken** [comment]
@jainendra001 there's no need to assign the Issue to you - The [linked file](https://gist.github.com/oliviaklohr/cb610c14ffc7f04336a622c7e7e16ec6) is 157 items and some will be on the main list already. 

I'd recommend using that list as a basis for contributions. As I mentioned before - one PR per company.

**Comment by ap0calypse9** [comment]
Is the Issue still available to woek on? Let me know

**Comment by dougaitken** [comment]
@ap0calypse9 I haven't compared [the file](https://gist.github.com/oliviaklohr/cb610c14ffc7f04336a622c7e7e16ec6) with the live site in some time but feel free to add contributions.

**Comment by pranay-sankar-muppanenei** [comment]
@dougaitken  i have added some company profiles like 3M,Mars  and so on after making sure they aren't present in the company folders.so can u please review my prs #2003, #2002

**Comment by dougaitken** [comment]
@pranay-sankar-muppanenei - please change the PR title since none of them alone close this Issue. Thanks.

**Comment by pranay-sankar-muppanenei** [comment]
Except the title,is the rest of PR  okay?Also @dougaitken  can u please suggest the appropriate title,I mean for every single company profile I am supposed to add a PR,but in the end  each PR adresses the parents issue .

**Comment by pranay-sankar-muppanenei** [comment]
@dougaitken  i have changed titles of PR's will this work.Also can you add please the hacktoberfest labels...

**Comment by dougaitken** [comment]
@pranay-sankar-muppanenei you need a README update for each PR to add the company to the list, not just the company profile.

**Comment by pranay-sankar-muppanenei** [comment]
@dougaitken I have added the README update for 3m-company PR  #2002. i will also create readme update for other pr's too,Thank you.

**Comment by dougaitken** [comment]
I'm closing this in favour of #2040

**Comment by pranay-sankar-muppanenei** [comment]
@dougaitken  is it possible to label hacktoberfest for every  company pull request that we create?

**Comment by dougaitken** [comment]
The repository is tagged so they will be counted.

**Comment by pranay-sankar-muppanenei** [comment]
@dougaitken okay, then first i will try to update the readme  for the other pull requests that i have created and then i will also try sending a pull requests for the companies listed in  #2040.
